serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -1 +1,202 @@
+For this guide, we use `JiB`. However, if you already use another extension or want to know how it works, please visit this {quarkus-container-images}[Quarkus' guide].
+
+The Kogito Examples do not include the needed extensions. We will add it using the Quarkus CLI:
+
+
+.Adding the JiB extension
+[source,shell]
+----
+$ quarkus extension add 'container-image-jib'
+----
+
+If you inspect the pom.xml, you can check the newly added dependency.
+
+.Checkin the JiB extension
+[source,xml,subs="attributes+"]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-container-image-jib</artifactId>
+</dependency>
+----
+
+=== Building the Serverless Workflow Application
+
+[source,shell]
+----
+$ quarkus build -Dquarkus.container-image.build=true \
+    -Dquarkus.container-image.group=kogito \
+    -Dquarkus.container-image.name=serverless-workflow-greeting-quarkus \
+    -Dquarkus.container-image.tag=1.0 \
+    -Dquarkus.container-image.registry=quay.io
+----
+
+Note that JiB caches the `target/lib` directory. Depending on how big your project is, it can speed up the rebuilds.
+If you want to push it to a registry, make sure to set the following properties accordingly to your needs:
+
+- quarkus.container-image.registry: registry address, e.g. _quay.io_.
+- quarkus.container-image.group: registry _context/namespace_ or if on Kubernetes or OpenShift the _namespace/project_.
+- quarkus.container-image.push: needs to be set to `true`, identify whether to push or not the image, defaults to `false`.
+
+.Example on pushing the built Application to Quay.io
+[source,shell]
+----
+$ quarkus build -Dquarkus.container-image.build=true \
+    -Dquarkus.container-image.group=kogito \
+    -Dquarkus.container-image.name=serverless-workflow-greeting-quarkus \
+    -Dquarkus.container-image.tag=1.0 \
+    -Dquarkus.container-image.registry=quay.io \
+    -Dquarkus.container-image.push=true
+----
+
+The command above will result in the following container image: `quay.io/kogito/serverless-workflow-greeting-quarkus:1.0`.
+
+Alternatively, you can create a maven profile to build the container image that can be triggered just by setting the target profile.
+
+.Maven Profile Example
+[source,xml]
+----
+<profile>
+  <id>build-container</id>
+  <properties>
+    <quarkus.container-image.build>true</quarkus.container-image.build>
+    <quarkus.container-image.group>kogito</quarkus.container-image.group>
+    <quarkus.container-image.name>serverless-workflow-greeting-quarkus</quarkus.container-image.name>
+    <quarkus.container-image.tag>1.0</quarkus.container-image.tag>
+    <quarkus.container-image.registry>quay.io</quarkus.container-image.registry>
+    <quarkus.container-image.push>true</quarkus.container-image.push>
+  </properties>
+</profile>
+----
+
+If using profiles, a new extension can be added to the target profile by setting profile name like the example below:
```

Comment:
```suggestion
If you are using profiles, a new extension can be added to the targeted profile by setting the profile name in the build command like the example below:
```

Reply:
This sentence should be rephrased, there is no support for adding extension with cli or quarkus maven plugin.